### PR TITLE
Add uSPAC10_JUN22 price identifier

### PR DESCRIPTION
**Pull request by UniSX-App**
This price identifier is absolutely the same as the previously approved uSPAC10 (UMIP-140) except for the list of shares and expiration date.

### CODE REVIEW

**Review by therealhexi — DISMISSED** [review]

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 1]
```diff
@@ -0,0 +1,452 @@
+## HEADERS
+|UMIP-      | | 
+|:-------------|:-----------------------------------------------------------------------------|
+|UMIP title|Add uSPAC10_JUN22 as price identifier|
+|Author|BinomFX (binomfx@gmail.com)|
+|Status|Draft| 
+|Created|28.03.2022|
+|Discourse Link|https://discourse.umaproject.org/t/add-uspac10-as-price-identifier-for-emp-contract/1357?u=binomfx|
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:8]: please update the link

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+
+## SUMMARY
+The DVM should support price requests for uSPAC10_MAR22 price identifier<br>
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:11]:
```suggestion
The DVM should support price requests for uSPAC10_JUN22 price identifier<br>
```

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+The purpose of these price identifier is to create synthetic token, price of which is linked to the value of index of **10** most active SPACs (Special Purpose Acquisition Companies) shares.<br> That synthetic token can be used for creating speculative strategies at IPO market.
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:12]:
```suggestion
The purpose of this price identifier is to create synthetic token, price of which is linked to the value of index of **10** most active SPACs (Special Purpose Acquisition Companies) shares.<br> That synthetic token can be used for creating speculative strategies at IPO market.
```

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 4]
```diff
@@ -0,0 +1,452 @@
...
+In addition, that token can be used as components associated with classical markets by other DeFi and DApp protocols, which makes it possible to scale.
+
+## TECHNICAL SPECIFICATION
+| | |
+|:---------------------------|:---------------------------------------------------|
+|**Identifier name**         |**uSPAC10_JUN22**|
+|Base asset                  | Most active SPAC shares.|
+|Quote Currency              | USD|
+|Intended Collateral Currency| USDC|
+|Expiration date             | 30.06.2022|
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:25]: If this is contract expiration then there is no specific need to duplicate it here as this is application specific parameter.

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 5]
```diff
@@ -0,0 +1,452 @@
...
+|2020|      248|           83,354.0|                336.1|
+|2019|       59|           13,608.3|                230.6|
+|2018|       46|           10,751.9|                233.7|
+|2017|       34|           10,048.5|                295.5|
+|2016|       13|            3,499.2|                269.2|
+
+By their nature, SPAC shares are subject to impulsive growth at the moment of information or even just rumors that a target company for a merger has been found.<br>
+`A good way to capitalize on such momentum growth without having to analyze hundreds of SPACs is to take advantage of the movement of the index value that includes stocks of the most active SPACs.`<br>
+
+The selection of **10** stocks of the most active SPACs included in the basket of the proposed uSPAC10 index is made according to [SPAC Analytics Top Performing SPACs](https://www.spacanalytics.com/).<br>
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:47]:
```suggestion
The selection of **10** stocks of the most active SPACs included in the basket of the proposed uSPAC10_JUN22 index is made according to [SPAC Analytics Top Performing SPACs](https://www.spacanalytics.com/).<br>
```

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 6]
```diff
@@ -0,0 +1,452 @@
...
+|Author|BinomFX (binomfx@gmail.com)|
+|Status|Draft| 
+|Created|28.03.2022|
+|Discourse Link|https://discourse.umaproject.org/t/add-uspac10-as-price-identifier-for-emp-contract/1357?u=binomfx|
+
+## SUMMARY
+The DVM should support price requests for uSPAC10_MAR22 price identifier<br>
+The purpose of these price identifier is to create synthetic token, price of which is linked to the value of index of **10** most active SPACs (Special Purpose Acquisition Companies) shares.<br> That synthetic token can be used for creating speculative strategies at IPO market.
+
+## MOTIVATION
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:14]: Please refer to uSPAC10 and what are key changes here.

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 7]
```diff
@@ -0,0 +1,452 @@
...
+Price requests should use the 1 minute quotes for the date corresponding to price request timestamp. Close price should be used.
+<br><br>
+##### Example "Stock Data – Rapidapi.com" request for 10 shares listed above **realtime prices**:
+```
+var axios = require("axios").default;
+
+var options = {
+  method: 'GET',
+  url: 'https://stock-data-yahoo-finance-alternative.p.rapidapi.com/v6/finance/quote',
+  params: {symbols: 'DWAC,IRDM,PRIM,TGLS,MP,LCID,GDYN,SMPL,KW,BWMX'},
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:87]: please update list of symbols for uSPAC10_JUN22 here and API response below

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 8]
```diff
@@ -0,0 +1,452 @@
...
+
+##### Example "Stock Data – Rapidapi.com" request for 10 shares listed above **historical** price:
+```
+var axios = require("axios").default;
+
+var options = {
+  method: 'GET',
+  url: 'https://stock-data-yahoo-finance-alternative.p.rapidapi.com/v8/finance/spark',
+  params: {
+    symbols: 'DWAC,IRDM,PRIM,TGLS,MP,LCID,GDYN,SMPL,KW,BWMX',
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:216]: please update list of symbols for uSPAC10_JUN22 here and API response below

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 9]
```diff
@@ -0,0 +1,452 @@
...
+If a request timestamp takes place on a weekend or any other day the stock market is closed, voters should use the latest tick as the price. For the weekend that would be the closing price of the asset on Friday and for official holidays this would be the last know price provided by the price feed.
+Please note that this is different than the normal calculation process, which requires using the open price of the period that the price request falls in.
+### Stock markets working hours
+Underlaying assets trade during exchange hours which leaves gaps in prices between 4:00PM EST close and 9:30AM EST open the next day and on weekends and market holidays.
+### Price feed
+Our price-feed provider’s API documentation can be found [here](https://rapidapi.com/principalapis/api/stock-data-yahoo-finance-alternative/).<br>
+A reference price feed implementation that is used by liquidator and dispute bots can be seen [here](__________________________________________________________)<br>
```
Comment (Reinis-FRP) [inline UMIPs/UMIP-uSPAC10_JUN22.md:385]: price feed implementation link is missing

**Review by Reinis-FRP — COMMENTED** [review]
please add only one UMIP file per PR
uSPAC5 and uSPAC_FR seem redundant here

File: UMIPs/UMIP-uSPAC10_JUN22.md (hunk 4 above)
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC10_JUN22.md:25]: ok

File: UMIPs/UMIP-uSPAC10_JUN22.md [hunk 10]
```diff
@@ -0,0 +1,452 @@
...
+|UMIP-      | | 
+|:-------------|:-----------------------------------------------------------------------------|
+|UMIP title|Add uSPAC10_JUN22 as price identifier|
+|Author|BinomFX (binomfx@gmail.com)|
+|Status|Draft| 
+|Created|28.03.2022|
+|Discourse Link|https://discourse.umaproject.org/t/add-uspac10-as-price-identifier-for-emp-contract/1357?u=binomfx|
+
+## SUMMARY
+The DVM should support price requests for uSPAC10_JUN22 price identifier<br>
```
Comment (smb2796) [inline UMIPs/UMIP-uSPAC10_JUN22.md:11]: Why is this specifically for JUN22?

File: UMIPs/UMIP-uSPAC10_JUN22.md (hunk 8 above)
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC10_JUN22.md:216]: done

File: UMIPs/UMIP-uSPAC10_JUN22.md (hunk 7 above)
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC10_JUN22.md:87]: done

File: UMIPs/UMIP-uSPAC5-perpetual.md [hunk 11]
```diff
@@ -0,0 +1,293 @@
...
+>Back in early 2018, MarketStack was initially presented under a different name with the aim of providing a free and cost-effective market data alternative to Yahoo Finance. In the course of the years, MarketStack REST API has become one of the most popular one-stop shop solutions for real-time, intraday and historical stock data, supporting a total of 170,000+ stock tickers from 70 global stock exchanges, including NASDAQ, Australian Stock Exchange, London Stock Exchange, and many more.
+
+>The MarketStack API is built on top of scalable, cutting-edge cloud infrastructure, handling millions of concurrent API requests each day. Using the API customer will be able to obtain rate information as well as metadata about stock tickers, companies as well as stock exchanges.
+
+Underlying stocks are traded during exchange hours which leaves gaps in prices between 4:00PM EST close and 9:30AM EST open the next day and on weekends and market holidays. 
+
+## IMPLEMENTATION
+### Price identifier
+#### 1. Read index basket
+1.1. Get IPFS address from smart contract.<br>
```
[the remark this answers is not in the document]
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC5-perpetual.md:73]: This is from UMIP for uSPAC5-perpetual. We're not doing this pricefeed right now. This is a different file even.

File: UMIPs/UMIP-uSPAC10_JUN22.md (hunk 6 above)
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC10_JUN22.md:15]: done

File: UMIPs/UMIP-uSPAC10_JUN22.md (hunk 10 above)
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC10_JUN22.md:11]: Added to UMIP: "The uSPAC10_JUN22 price list is based on the uSPAC10 price list and differs from it only by the list of stocks included in the index basket."

File: UMIPs/UMIP-uSPAC5-perpetual.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (smb2796) [reply UMIPs/UMIP-uSPAC5-perpetual.md:73]: Ok, it needs to be removed from the pr then

File: UMIPs/UMIP-uSPAC10_JUN22.md (hunk 1 above)
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC10_JUN22.md]: done

File: UMIPs/UMIP-uSPAC5-perpetual.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (UniSX-App) [reply UMIPs/UMIP-uSPAC5-perpetual.md]: uSPAC5-perpetual deleted

**Pull request closed without merging**